src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceInObject = {};
+
+  sourceString.split(';').forEach(element => {
+    const elementSplit = element.split(':');
```

Comment:
Read [checklist point 4](https://github.com/mate-academy/js_style-to-object/blob/master/checklist.md)
```suggestion
  sourceString
    .split(';')
    .forEach(element => {
      const elementSplit = element.split(':');
```